scripts/cf_deploy.sh

```diff
@@ -75,6 +75,21 @@ unbind_all() {
   unbind $BACKEND_APP_NAME
 }
 
+binding_update() {
+  app_name=$1
+  sub_dir=$2
+
+  app_guid=$(cf app "$app_name" --guid)
+  echo "app guid for $app_name: $app_guid"
+  gateway_service_instance_id="$(cf service $GATEWAY_NAME --guid)"
+  gateway_url=$(cf curl /v2/service_instances/"$gateway_service_instance_id" | jq .entity.dashboard_url | sed "s/\/scg-dashboard//" | sed "s/\"//g")
+
+
+  status_code=$(curl -k -XPUT "$gateway_url/actuator/bound-apps/$app_guid/routes" -d "@$ROOT_DIR/$sub_dir/gateway-config.json" \
+       -H "Authorization: $(cf oauth-token)" -H "Content-Type: application/json" --write-out %{http_code}  --silent --output /dev/null)
```

Comment:
Do we want the output to be silent in this case? I have had situations where the output of the response was useful beyond the status code.

Reply:
👍 Updated the output in the latest commit.